ui/views/mus/desktop_window_tree_host_mus.cc

```diff
@@ -219,6 +219,11 @@ DesktopWindowTreeHostMus::DesktopWindowTreeHostMus(
   // Widget. This call enables that.
   NativeWidgetAura::RegisterNativeWidgetForWindow(desktop_native_widget_aura,
                                                   window());
+
+#if defined(USE_OZONE) && !(OS_CHROMEOS)
```

Comment:
for consistency, I believe we are using if os_linux && ozone && !os_chromeos

in other places.

Reply:
done.